bin/scurl

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
+curl --proto '=https' --tlsv1.2 -sSfL "$@"
```

Comment:
```suggestion
exec curl --proto '=https' --tlsv1.2 -sSfL "$@"
```

we can exec here, which basically discards the shell process instead of keeping it. super minor